omocha-api/src/main/java/org/omocha/api/interfaces/mapper/QuestionDtoMapper.java

```diff
@@ -0,0 +1,47 @@
+package org.omocha.api.interfaces.mapper;
+
+import java.util.List;
+import java.util.stream.Collectors;
+
+import org.mapstruct.InjectionStrategy;
+import org.mapstruct.Mapper;
+import org.mapstruct.ReportingPolicy;
+import org.omocha.api.interfaces.dto.QuestionDto;
+import org.omocha.domain.auction.qna.QuestionCommand;
+import org.omocha.domain.auction.qna.QuestionInfo;
+import org.springframework.data.domain.Page;
+import org.springframework.data.domain.PageImpl;
+
+@Mapper(
+	componentModel = "spring",
+	injectionStrategy = InjectionStrategy.CONSTRUCTOR,
+	unmappedTargetPolicy = ReportingPolicy.ERROR
+)
+public interface QuestionDtoMapper {
+
+	QuestionCommand.ModifyQuestion toCommand(Long memberId, Long questionId,
+		QuestionDto.ModifyQuestionRequest modifyQuestionRequest);
```

Comment:
이것도 아래처럼 개행을 해주시면 보기 좋을 것 같네요!
```suggestion
	QuestionCommand.ModifyQuestion toCommand(
        Long memberId, 
        Long questionId,
		QuestionDto.ModifyQuestionRequest modifyQuestionRequest
    );
```